checker/manual/troubleshooting.tex

```diff
@@ -741,10 +736,30 @@ \subsection{Build the Checker Framework\label{building}}
 \end{Verbatim}
 % $ to unconfuse Emacs LaTeX mode
 
-\item Add \code{tools.jar} and \code{checker.jar} to your classpath.
-  (If you do not do this, you will have to supply the \code{-cp} option
-  whenever you run \code{javac} and use a checker plugin.)
-  Example command:
+Once \<checker.jar> is built, you may wish to put the Checker Framework \<javac> even earlier on
```

Comment:
Under what circumstances might one wish to do that?  The manual should say, to help readers understand whether this section is relevant to them.